import/README.md

```diff
@@ -1,26 +1,48 @@
 # Import
 
-In this directory you'll find the best strategies to import CSV and Cypher queries into Memgraph.
+In this directory, you'll find the best strategies for importing CSV and Cypher queries into Memgraph.
 
+The cypher and csv folders contain examples of how to load a CYPHERL file and LOAD CSV via pymgclient and neo4jpython client. 
 
-## Node import speed reference
+Before running any import script, make sure the `../dataset/graph500/download_dataset.sh` is executed first, 
```

Comment:
Link to the script or instructions on how to run it

Reply:
Added